plugin-build/plugin/src/main/java/com/ncorti/ktfmt/gradle/tasks/CreateGitPreCommitHookTask.kt

```diff
@@ -0,0 +1,20 @@
+package com.ncorti.ktfmt.gradle.tasks
+
+import com.ncorti.ktfmt.gradle.tasks.worker.PreCommitHookFormatAction
+import javax.inject.Inject
+import org.gradle.api.tasks.SourceTask
+import org.gradle.api.tasks.TaskAction
+import org.gradle.workers.WorkerExecutor
+
+/** ktfmt-gradle Git hook task. Creates a git hook that runs ktfmt on git commit */
+abstract class CreateGitPreCommitHookTask : SourceTask() {
```

Comment:
```suggestion
abstract class CreateGitPreCommitHookTask : DefaultTask() {
```

@gpopides  This is the cause of the Windows failure. This is not a Source Task as you're not populating the `source` property. Windows is failing while attepting to MD5 the source field which is null here

Reply:
Thanks a lot for the feedback, i will change that.